crates/libcruby-sys/src/lib.rs

```diff
@@ -92,4 +92,7 @@ extern "C" {
                       arg: void_ptr,
                       state: *mut RubyException)
                       -> VALUE;
+
+    pub fn rb_str_new_cstr(string: c_string) -> VALUE;
+    pub fn rb_utf8_str_new(string: c_string, len: libc::c_long) -> VALUE;
```

Comment:
Sure I will try re-exporting in helix_runtime instead. I'm not too familiar with MRI, so I don't know if there is a difference.